Makefile

```diff
@@ -78,6 +78,9 @@ test-python-universal-local:
 test-python-universal:
 	FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 --integration --universal sdk/python/tests
 
+test-python-go-server:
```

Comment:
Is the idea that this command would be run if you are developing Go code? (as the most comprehensive way to test Go retrieval)

Reply:
To your first question, yes, but I think we should add the configs for this test to the universal tests once ODFVs are supported for the go server? @felixwang9817 may be working on adding this command to CI?